linux.md

```diff
@@ -198,10 +198,9 @@ Make sure that you're running the 2nd-newest OS version or the newest version -
     wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
     sudo apt update
     sudo apt install postgresql-16
-    echo "export PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bashrc'`
+    echo -e "export PSQL_PAGER=\"less --chop-long-lines --header 1\"\nexport LANG=en_US.UTF-8" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bashrc'`
```

Comment:
1. Add a setting for the environment variable for the `LANG` locale, as Postgres server will use the encoding set to it and ignore the settings in the `postgresql.conf` file